tms/clickup/views.py

```diff
@@ -62,6 +63,22 @@ def post(self, request):
         }, status=status.HTTP_201_CREATED)
 
 
+class LogoutView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        try:
+            if request.user.is_authenticated:
```

Comment:
Here you already add permission_class so you don't have to check request.user is authenticated.

Reply:
please check last commit this already change